flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveTableSink.java

```diff
@@ -337,6 +340,12 @@ private DataStreamSink<Row> createBatchSink(
         builder.setTempPath(
                 new org.apache.flink.core.fs.Path(toStagingDir(stagingParentDir, jobConf)));
         builder.setOutputFileConfig(fileNaming);
+        builder.setIdentifier(identifier);
+        builder.setPolicyKind(conf.get(HiveOptions.SINK_PARTITION_COMMIT_POLICY_KIND));
+        builder.setCustomClass(
+                conf.get(FileSystemConnectorOptions.SINK_PARTITION_COMMIT_POLICY_CLASS));
+        builder.setSuccessFileName(
+                conf.get(FileSystemConnectorOptions.SINK_PARTITION_COMMIT_SUCCESS_FILE_NAME));
```

Comment:
Please only options of `HiveOptions` in the Hive connector. Otherwise, it's error-prone to use `FileSystemConnectorOptions`, e.g., unexpected default value. You can reuse the options by referring the definition from `FileSystemConnectorOptions`, for example, in `HiveOptions.java`:

```java
public static final ConfigOption<String> SINK_PARTITION_COMMIT_POLICY_CLASS =
            FileSystemConnectorOptions.SINK_PARTITION_COMMIT_POLICY_CLASS;

    public static final ConfigOption<String> SINK_PARTITION_COMMIT_SUCCESS_FILE_NAME =
            FileSystemConnectorOptions.SINK_PARTITION_COMMIT_SUCCESS_FILE_NAME;
```